apps/marginfi-v2-ui/src/components/common/Wallet/components/AuthenticationV2/AuthDialog.tsx

```diff
@@ -141,7 +141,7 @@ export const AuthDialog = () => {
 
   return (
     <div>
-      <Progress value={progress} className="fixed top-0 z-[999] h-1 rounded-none" />
+      <Progress value={progress} className={`fixed top-0 h-1 rounded-none ${isWalletAuthDialogOpen && "z-[999]"}`} />
```

Comment:
thinking about it, we should just conditionally render the whole component, rather than just changing zindex. it shouldn't be rendered at all when dialog is closed.